kernel/classes/datatypes/ezimage/ezimagetype.php

```diff
@@ -190,6 +190,7 @@ function deleteStoredObjectAttribute( $contentObjectAttribute, $version = null )
             $imageHandler->setAttribute( 'alternative_text', false );
             $imageHandler->removeAliases();
             $imageHandler->store( $contentObjectAttribute );
+            $contentObjectAttribute->setContent( null );
```

Comment:
Nice, thanks! Looks like the proper way to fix this. I'm only wondering why you're nulling the content _after_ the imagehandler store call? Wouldn't before make more sense?

Reply:
Everything was stored properly as far as I could see. The issue was the content object attribute object in memory because of the "objects are always references"-thing in PHP (at least that what my "hunch" said)